packages/edit-site/src/components/navigation-sidebar/navigation-panel/style.scss

```diff
@@ -16,9 +21,15 @@
 	position: relative;
 	width: $nav-sidebar-width;
 	height: 100%;
+	padding-top: $header-height;
+	overflow: hidden;
 }
 
 .edit-site-navigation-panel__site-title-container {
+	position: absolute;
+	top: 0;
+	left: 0;
```

Comment:
Maybe I'm missing something, but we wouldn't need this positioning if we didn't add the `padding-top` to `.edit-site-navigation-panel__inner`. 🤔

Reply:
Yep! Unnecessary. That was an artifact from a previous solution, thanks for noticing!